exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = 16 > 9; // finish this statement
 
 var roomMaxCapacity = 25;
-var enoughSpaceInRoom; // finish this statement
+var enoughSpaceInRoom = 25 >= studentCount + mentorCount; // finish this statement
```

Comment:
This is all good logic, but you could improve this by referring to the saved variables.

moreStudentsThanMentors = studentCount > mentorCount
enoughSpaceInRoom = roomMaxCapacity >=  studentCount + mentorCount

This way, if you ever need to update a number like maxCapacity then you only have to change it in one place.

Reply:
Yeah that's a good point.